Cargo.toml

```diff
@@ -42,6 +42,8 @@ strip = false
 
 [dependencies]
 argon2 = "0.5.2"
+async-graphql = { version = "6", features = ["dynamic-schema"] }
```

Comment:
Now that it officially goes to main, we should probably set a version (as it can be released anytime)